source/docs/reference/configurations.md

```diff
@@ -130,25 +130,44 @@ module.exports = ["products", "categories", "pages"];
 
 ### `config/caching.js`
 
-Allows to define configurations related to dataloader caching and implementation
+Allows to define configurations related to dataloader caching and implementations. It can export a configuration with the following keys:
 
-<blockquote class="wip">
-**Work In Progress:** more details about specific configurations will be detailed as part of https://github.com/front-commerce/developers.front-commerce.com/issues/49. If you need it right away, please [contact us](mailto:contact@front-commerce.com). We will make sure to answer you in a timely manner.
+* `defaultMaxBatchSize`: default batch size used for dataloaders (unless specified during instantiation) (default: `100`)
+* `strategies`: list of caching strategies to use in the application (default: `[]`)
+
+Each strategy can be configured with the keys below:
+
+* `implementation`: name of the [implementation of this strategy](/docs/advanced/graphql/dataloaders-and-cache-invalidation.html#Caching-strategies) (**mandatory**)
+* `supports`: list of loaders impacted by this strategy. Either an array of values or `"*"` for all. (default: `[]`)
```

Comment:
Wouldn't `*` be a better default here? Or should it be mandatory? Setting up a strategy that does nothing does not seem useful.

Reply:
That is something I asked myself during the implementation, but I was even more afraid by the impacts of adding a caching strategy to **ALL** dataLoaders by mistake (e.g a typo with `support` instead of `supports`).

Maybe we should make this a mandatory parameter instead, as I agree that it makes no sense to have something that does nothing by default.

Reply:
Let's take the mandatory route then. :)

Reply:
Done in https://gitlab.com/front-commerce/front-commerce/-/merge_requests/188.
Documentation updated by anticipation.